jni/lib/src/jprimitives.dart

```diff
@@ -2,117 +2,122 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-part of 'types.dart';
+// The types here are mapped to primitive types in Java, so they're all in
```

Comment:
If this is documentation for the whole file, consider using

```dart
/// bla
library;
```

Reply:
Requires to update the SDK to 2.19 to use unnamed libraries. I'll do it when we actually want to use other Dart 3 features in the future.